forLoopTimeout/forLoopTimeout.js

```diff
@@ -11,3 +11,35 @@ for (var i = 1; i <= 10; i++) {
 		console.log(i);
 	}, 0);
 }
+
+/*
+ The variable i is being passed by reference into setTimeout and not the actual value. After 10 seconds the value of i has already been 
+ incremented to 11 and passed to console.log().
+ The function only executes one time after the provided time interval expires and by that time the i value has already finished it's increments which causes setTimeout to be passed 11.
+*/
+
+
+/* 
+
+  Fix 1
+  setTimeout() needs to receive the actual value of i and not the reference after each iteration of the for loop
+  
+  for (var i = 1; i <= 10; i++) {
+    setTimeout(function(i) { 
```

Comment:
Nailed It! One caveat is that I like to use fat arrow functions in my set timeouts, (ESPECIALLY when working in "this" style of coding). Because they provide a block scoped environment and allow closure to be worked with more concisely. You're solution is great here and works well, I just think that one of the BEST use cases for the fat arrow function is actually being provided as the callback to setTimeout. That piece of knowledge may help you later on, but take it at face value!